workspaces/ai-integrations/app-config.yaml

```diff
@@ -91,8 +91,10 @@ catalog:
   rules:
     - allow: [Component, System, API, Resource, Location]
   providers:
-    modelCatalog:
-      # The field underneath here can be any value that you feel represents your model catalog bridge instance
-      development:
-        # If testing locally, this value should be set to `localhost:9090`
-        baseUrl: '${RHDH_AI_BRIDGE_SERVER}'
+   modelCatalog:
```

Comment:
Indentation looks off from the previous config (and prettier in the CI seems to be complaining about it)

Reply:
ok @johnmcollier I think I have it sorted out now ... ptal